verification.py

```diff
@@ -4087,6 +4087,285 @@ def test_simulate_from_demography(self):
             pyplot.close("all")
 
 
+class MutationStatsTest(Test):
+
+    p_threshold = 0.001
+
+    def chisquare(self, observed, expected, p_th=p_threshold):
+        obs = observed.reshape(np.product(observed.shape))
+        exp = expected.reshape(np.product(observed.shape))
+        assert np.all(obs[exp == 0] == 0)
+        not_zeros = exp > 0
+        if sum(not_zeros) > 1:
+            chisq = scipy.stats.chisquare(obs[not_zeros], exp[not_zeros])
+            assert chisq.pvalue > p_th
```

Comment:
What happens if this assertion fails? Does it bring everything down with a clear error message when running with multiple threads? I just want to be sure that I don't miss a problem if this ever fails.

Reply:
Whoops - didn't even mean to leave this in here. I could put these p-value checks back in, but they *should* be redundant with the figures.

Reply:
It's up to you - might be best to remove them and keep things simple.